src/app/components/goal-management/goal-management.component.ts

```diff
@@ -8,8 +11,16 @@ import {FormControl} from '@angular/forms';
 })
 export class GoalManagementComponent implements OnInit {
   date = new FormControl(new Date());
+  goalSet: Goal[];
```

Comment:
Looks like this is a list but you are naming it a set. Please name if properly.

Reply:
Ok, change to goalList